src/types/errors/selling-partner-api-errors.ts

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable max-classes-per-file */
+import toNumber from 'lodash.tonumber'
```

Comment:
Why do we need this? Why not just do `Number("123")` or `parseFloat("123.456")`?